File: CS/TreeListCellMerging/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraTreeList;
using DevExpress.XtraTreeList.Nodes;
using DevExpress.XtraTreeList.Columns;

namespace TreeListCellMerging
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }



     
        private void Form1_Load(object sender, EventArgs e)
        {
            this.departmentsTableAdapter.Fill(this.departmentsDataSet.Departments);
            treeList1.ExpandAll();
        }
    }
}

File: VB/TreeListCellMerging/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraTreeList
Imports DevExpress.XtraTreeList.Nodes
Imports DevExpress.XtraTreeList.Columns

Namespace TreeListCellMerging
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub




		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Me.departmentsTableAdapter.Fill(Me.departmentsDataSet.Departments)
			treeList1.ExpandAll()
		End Sub
	End Class
End Namespace